koku/masu/openapi.json

```diff
@@ -372,39 +372,49 @@
         },
         "parameters": []
       },
-      "/status/": {
-        "get": {
-          "operationId": "status_list",
-          "description": "Packages response for class-based view.",
-          "parameters": [
-            {
-              "name": "celery",
-              "in": "query",
-              "description": "Whether to return celery queue/task data",
-              "required": false,
-              "schema": {
-                "type": "boolean",
-                "example": true
+    "/status/": {
+      "get": {
+        "operationId": "getStatus",
+        "description": "Packages response for class-based view.",
+        "responses": {
+          "200": {
+            "description": "A JSON object containing pet information",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/StatusResponse"
+                }
               }
             }
-          ],
-          "responses": {
-            "200": {
-              "description": "",
-              "content": {
-                "application/json": {
-                  "schema": {
-                    "$ref": "#/components/schemas/StatusResponse"
-                  }
+          }
+        },
+        "tags": [
+          "Status"
+        ]
+      },
+      "parameters": []
+    },
+    "/celery-status/": {
```

Comment:
Wondering whether we should split this out since as far as I can tell you can't have multiple responses returned based from passed in parameters. The response from /status/?celery=true is completely different to /status/

`
{
    "scheduled_count": 0,
    "reserved_count": 0,
    "active_count": 0
}`